nwtopt.sub

```diff
@@ -13,10 +13,10 @@ error                   = logs/errors/errors_$(CLUSTER)_$(PROCESS).txt
 stream_output           = True
 stream_error            = True
 should_transfer_files   = Yes
-arguments               = <ip>:<port>/db 400
+arguments               = 130.11.177.99:27017/db 240
```

Comment:
A couple things 0) not good to have an internal IP address in a public repo and 1) don't you want this the way you had it so that users specify IP and PORT per submit?

Reply:
ok - now it seems this got replaces when you ran. Consider making a template version (like `nwtopt.submit.template` )with the placeholders for <ip> and <port> and reading that with the code which then writes the actual sub file?

Reply:
Yeah this was a mistake in the commit, I'll revert it back (thought I had .gitignore not tracking the file, but apparently it is??)